docker/windows/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM microsoft/windowsservercore:1709 AS base
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV PYTHON_VERSION 3.6.5
+ENV NODEJS_VERSION 8.11.1
+ENV DOTNETCORESDK_VERSION 2.1.4
+ENV DOCKER_VERSION 17.09.0
```

Comment:
Can you also add Python 2.7?

Reply:
Yes, but it will be complex since Python 2.7 and Python 3.6 use two different installers :(. I'd like to be able two support the two versions without having to maintain also two dockerfiles

Reply:
Yep, make sense